espanol/agasajo/agasajo.md

```diff
@@ -0,0 +1,2 @@
+El cocinero huyó sin dar explicaciones.
+[El cocinero fue a controlar si tenia suficiente comida. Abrio la despensa y se dio cuenta enseguida que..](..comida/comida.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.